src/backoff.erl

```diff
@@ -1,13 +1,20 @@
 -module(backoff).
 -export([increment/1, increment/2]).
+-export([rand_increment/1, rand_increment/2]).
+-export([jitter/1]).
 -export([init/2, init/4,
          fire/1, get/1, succeed/1, fail/1]).
+
 -record(backoff, {start :: pos_integer(),
                   max :: pos_integer() | infinity,
                   current :: pos_integer(),
+                  jitter=false :: boolean(),
                   value :: term(),
                   dest :: pid()}).
 
+-define(RAND_FACTOR, 0.5).
+-define(DEF_MULTIPLIER, 1.5).
```

Comment:
What led to choosing these values? I might like a comment there. Is it worth having them be tweakable via env config?